contracts/Chaosnet.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity 0.8.9;
+
+/// @title Chaosnet
+/// @notice This is a beta staker program for stakers willing to go the extra
+/// mile with monitoring, share their logs with the dev team, and allow to more
+/// carefully monitor the bootstrapping network. As the network matures, the
```

Comment:
`bootstrapping` - Just to confirm this is not about the bootstrap nodes only, but about the early stage of the network, right?

Reply:
Yes, this is for the early stage of the network, aka bootstrapping the network.